dlr/dlr-engine/src/test/java/ai/djl/dlr/integraion/DlrTest.java

```diff
@@ -40,10 +41,11 @@ public void testDlr() throws ModelException, IOException, TranslateException {
         String os;
         if (System.getProperty("os.name").toLowerCase().startsWith("mac")) {
             os = "osx";
-        } else if (System.getProperty("os.name").toLowerCase().startsWith("linux")) {
+        } else if (System.getProperty("os.name").toLowerCase().startsWith("linux")
```

Comment:
Why we fail on GPU, if the DLR doesn't GPU, it should fallback to CPU.
It should controlled by getCapadatability() call in the DlrEngine.

Does DLR runtime has to match with the model?
If DLR runtime support GPU, can it run CPU model?
Does that mean if we run one CPU model and one GPU model, we have to load two DLR runtime?